tests/package/test.sc

```diff
@@ -27,14 +27,14 @@ trait FailureWatcher extends Suite {
 }
 
 trait MesosTest extends UnitTest with BeforeAndAfterAll with FailureWatcher {
-  val packagePath = pwd / RelPath("..") / RelPath("..") / 'target / 'packages
-  val PackageFile = "^([^-]+).+?\\.(rpm|deb)$".r
+  val packagePath = pwd / up / up / 'tools / 'packager
+  val PackageFile = "^marathon_.+\\.([a-z0-9]+)_all.(rpm|deb)$".r // TODO: match rpm packages as well.
```

Comment:
This is not quite working. I'm not sure it if has to be that complex though. What do you think, @timcharper?